components/fieldgroups/profile/marketo-engage-person-details.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/marketo-engage-person-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Engage Person Details",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This field group is used to capture Marketo Engage cookie IDs for the B2B Persons.",
+  "definitions": {
+    "marketo-engage-person-details": {
+      "properties": {
+        "xdm:marketoEngage": {
+          "type": "object",
+          "properties": {
+            "xdm:cookieIDs": {
```

Comment:
Can you explain the use case for storing the cookie ID in a profile fragment, since this class is intended to extend profile class?

Reply:
Hi, yes, our product owner has written this document that describes the purpose: https://wiki.corp.adobe.com/x/8g0Utg